spec/avro_spec.rb

```diff
@@ -3,7 +3,9 @@
 describe AvroTurf do
   include FakeFS::SpecHelpers
 
-  let(:avro) { AvroTurf.new(schemas_path: "spec/schemas/") }
+  let(:log) { StringIO.new }
+  let(:logger) { Logger.new(log) }
+  let(:avro) { AvroTurf.new(schemas_path: "spec/schemas/", logger: logger) }
```

Comment:
The schemas are "written" on a per-example basis, according to the needs of that example. `spec/schemas/` is the base directory – in some examples, we want to verify behavior when there's a difference between the writer's schema and the reader's schema, so we use two different base directories.